src/engine/channels/enginedeck.cpp

```diff
@@ -7,8 +7,15 @@
 #include "engine/enginepregain.h"
 #include "engine/enginevumeter.h"
 #include "moc_enginedeck.cpp"
+#include "track/track.h"
 #include "util/sample.h"
 
+#ifdef __STEM__
+namespace {
+constexpr int kMaxSupportedStem = 4;
```

Comment:
`kMaxSupportedStem` is used everywhere else in the code. I used to have an instance of `kMaxStemCount` but I replaced it because i thought `kMaxSupportedStem` had a better meaning. Happy to replace them all if you thing `kMaxStemCount` sounds best.

Reply:
The main reason I have stumbled here was the missing plural s. If you add that, I am fine.